### PR TITLE
Fix a typo where wrong function is called in example

### PR DESCRIPTION
**Pull request by arthurborisow**
Just a small typo

### CODE REVIEW

File: src/Chapter1.hs [hunk 1]
```diff
@@ -613,7 +613,7 @@ Implement a function that returns the sum of the last two digits of a number.
 6
 >>> sumLast2 134
 7
->>> sumLast 1
+>>> sumLast2 1
```
Comment (chshersh) [inline src/Chapter1.hs:616]: @arthurborisow Thanks for the fix! Would it be for hard for you to also fix the type of this comment? Just replace

```haskell
{-
=⚔️= Task 9
```

with

```haskell
{- |
=⚔️= Task 9
```

few lines above. Doctest tests only functions inside special comments (with `|` after `{-`), and this part was missing for a comment in this task. That's why we didn't notice this error before.

**Review by chshersh — APPROVED** [review]

File: src/Chapter1.hs (hunk 1 above)
Comment (arthurborisow) [reply src/Chapter1.hs:616]: Sure, no problem!

Comment (arthurborisow) [reply src/Chapter1.hs:616]: @chshersh Another thing that I noticed: `Now it is time to the open pull request with your changes` - `the` part is redundant I believe

Comment (vrom911) [reply src/Chapter1.hs:616]: Yes, nice catch, @arthurborisow! You can add it to this PR as well, if you don't mind 🙂

Comment (arthurborisow) [reply src/Chapter1.hs:616]: Done

**Comment by chshersh** [comment]
@arthurborisow Many thanks! :hugs:

**Pull request merged**